asb-ballerina/types.bal

```diff
@@ -128,6 +129,8 @@ public type ASBServiceReceiverConfig record {
     string connectionString;
     @display {label: "Entity Configuration"}
     TopicSubsConfig|QueueConfig entityConfig;
+    @display {label: "Dead letter queue"}
```

Comment:
Convention hasn't been followed. "Dead letter queue" -> "Dead Letter Queue"

Reply:
Fixed with the naming convention